paddle/framework/tensor_test.cc

```diff
@@ -260,3 +260,42 @@ TEST(Tensor, CopyFrom) {
   }
 #endif
 }
+
+namespace paddle {
+namespace framework {
+
+class LODTensorTester : public ::testing::Test {
+ public:
+  virtual void SetUp() override {
+    lod_tensor = decltype(lod_tensor)(new LODTensor);
+    // tensor's batch_size: 30
+    // 3 levels
+    // 0 10 20
+    // 0 5 10 15 20
+    // 0 2 5 7 10 12 15 20
+    auto lod = std::make_shared<LODTensor::lod_t>();
+    lod->emplace_back(LODTensor::level_t{0, 10, 20});
```

Comment:
```cpp
auto lod = std::make_shared<LODTensor::lod_t>({
  {0, 10, 20},
  {0, 5, 10, 15, 20},
  {0, 2, 5, 7, 10, 12, 15, 17, 20}
});
```

I am not sure which is more readable, but C++ 11 make initialization of an std container very easy.